recipes/sos-notebook/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "sos-notebook" %}
+{% set version = "0.19.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 91fc849a9d8c0b5fb975ec82eb1f2b9be2603ff936b8241a928cc8d738a57e16
+
+build:
+  skip: True  # [win]
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - {{ PYTHON }} -m sos_notebook.install --sys-prefix
+
+requirements:
+  host:
+    - ipython
+    - notebook >=5.0.0
+    - pip
+    - python >=3.6
```

If this can't be `noarch` you should add `skip: True  # [py<36]` to the `build` section and specify remove the version constraint from Python:
```suggestion
    - python
```
(same applies in the `run` section)

---

Having looked more closely it looks like this could be `noarch: python`?